server/routes.js

```diff
@@ -14,6 +14,18 @@ emailRewrite = require('./lib/email_rewrite.js'),
       secLog = require('./lib/security_logging'),
          url = require('url');
 
+var versionData;
+try {
+  versionData = require('../version.json');
+} catch(e) {
+  versionData = {
+    commit: "not found",
+    source: "not found",
+    version: "not found",
+    build: "not found"
+  };
+}
+
```

Comment:
yeah. Alternatively, could set `versionData = { commit: "not found", ... }`, and then try the require. A failed require leaves the default 'not found' still set. But maybe not a "better" anyways.